lang/cs/Org.Apache.REEF.Client/Common/DriverFolderPreparationHelper.cs

```diff
@@ -154,8 +154,18 @@ internal void CreateDefaultFolderStructure(AppParameters appParameters, string d
             File.WriteAllText(Path.Combine(driverFolderPath, _fileNames.GetBridgeExeConfigPath()), config);
 
             // generate .config file for Evaluator executable
-            File.WriteAllText(Path.Combine(driverFolderPath, _fileNames.GetGlobalFolderPath(), EvaluatorExecutable), 
-                DefaultDriverConfigurationFileContents);
+            var userDefinedEvaluatorConfigFile = Path.Combine(JarFolder, EvaluatorExecutable);
```

Comment:
Shouldn't there be a `.config` added to this file name? Also, as this isn't a file object, please rename to `*FileName`.

Reply:
EvaluatorExecutable already contains .config. 
Yeah, changed variable name.